src/OpenSage.Game/Gui/DebugUI/DebugMessageHandler.cs

```diff
@@ -7,7 +7,7 @@ public class DebugMessageHandler : InputMessageHandler
     {
         private readonly DebugOverlay _overlay;
 
-        public override HandlingPriority Priority => HandlingPriority.UIPriority;
+        public override HandlingPriority Priority => HandlingPriority.DebugPriority;
```

Comment:
The function keys (F2-F5) would get handled by the WndInputMessage handler ocassionally, because they had the same priority. It would only function randomly